app/modules/session/webauthn.server.ts

```diff
@@ -0,0 +1,145 @@
+import {
+  generateAuthenticationOptions,
+  generateRegistrationOptions,
+  verifyAuthenticationResponse,
+  verifyRegistrationResponse,
+} from '@simplewebauthn/server';
```

Comment:
These functions need to be in this `.server.ts` file and on in our route files to prevent Remix from throwing an error.